Comparator.dcm

```diff
@@ -80,187 +80,187 @@ $ENDCMP
-K dual comparator
+K cmp
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6561-1R-1U-2-4-1.8V-Low-Power-Push-Pull-Output-Comparator-DS20002139E.pdf
 $ENDCMP
 #
 $CMP MCP6561x-LT
 D Single 1.8V Low-Power Push-Pull Output Comparator, SC-70-5
-K single comparator
+K cmp
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6561-1R-1U-2-4-1.8V-Low-Power-Push-Pull-Output-Comparator-DS20002139E.pdf
 $ENDCMP
 #
 $CMP MCP6562
 D Dual 1.8V Low-Power Push-Pull Output Comparator, SOIC-8/MSOP-8
-K dual comparator
+K cmp
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6561-1R-1U-2-4-1.8V-Low-Power-Push-Pull-Output-Comparator-DS20002139E.pdf
 $ENDCMP
 #
 $CMP MCP6566
 D Single 1.8V Low-Power Open-Drain Output Comparator, SOT-23-5/SC-70
-K single comparator
+K cmp collector
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6566-6R-6U-7-9-1.8V-Low-Power-Open-Drain-Output-Comparator-DS20002143G.pdf
 $ENDCMP
 #
 $CMP MCP6566R
 D Single 1.8V Low-Power Open-Drain Output Comparator, SOT-23-5
-K single comparator
+K cmp collector
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6566-6R-6U-7-9-1.8V-Low-Power-Open-Drain-Output-Comparator-DS20002143G.pdf
 $ENDCMP
 #
 $CMP MCP6566U
 D Single 1.8V Low-Power Open-Drain Output Comparator, SOT-23-5
-K single comparator
+K cmp collector
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6566-6R-6U-7-9-1.8V-Low-Power-Open-Drain-Output-Comparator-DS20002143G.pdf
 $ENDCMP
 #
 $CMP MCP6567
 D Dual 1.8V Low-Power Open-Drain Output Comparator, SOIC-8/MSOP-8
-K dual comparator
+K cmp collector
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6566-6R-6U-7-9-1.8V-Low-Power-Open-Drain-Output-Comparator-DS20002143G.pdf
 $ENDCMP
 #
 $CMP MCP6569
 D Quad 1.8V Low-Power Open-Drain Output Comparator, SOIC-14/TSSOP-14
-K quad comparator
+K cmp collector
 F http://ww1.microchip.com/downloads/en/DeviceDoc/MCP6566-6R-6U-7-9-1.8V-Low-Power-Open-Drain-Output-Comparator-DS20002143G.pdf
 $ENDCMP
 #
 $CMP MCP65R41
 D 3 μA Comparator with Integrated Reference Voltage, SOT-23-6
-K comparator reference
+K cmp reference
 F http://ww1.microchip.com/downloads/en/DeviceDoc/22269B.pdf
 $ENDCMP
 #
 $CMP MCP65R46
 D 3 μA Comparator with Integrated Reference Voltage, SOT-23-6
-K comparator reference
+K cmp reference open collector drain
 F http://ww1.microchip.com/downloads/en/DeviceDoc/22269B.pdf
 $ENDCMP
 #
 $CMP MIC845H
 D Micro-Power Comparator / Battery Monitor, Active-High Push-Pull Output, SC-70-5
-K single comparator battery monitor
+K single cmp
 F http://ww1.microchip.com/downloads/en/DeviceDoc/mic845.pdf
 $ENDCMP
 #
 $CMP MIC845L
 D Micro-Power Comparator / Battery Monitor, Active-Low Push-Pull Output, SC-70-5
-K single comparator battery monitor
+K single cmp
 F http://ww1.microchip.com/downloads/en/DeviceDoc/mic845.pdf
 $ENDCMP
 #
 $CMP MIC845N
 D Micro-Power Comparator / Battery Monitor, Active-High Open-Drain Output, SC-70-5
-K single comparator battery monitor
+K single cmp collector
```

Comment:
this has the single keyword, where other have the dual/quad keyword removed.

Reply:
'single' not already in description